index.html

```diff
@@ -144,25 +156,20 @@
             </div>
           </div>
         </form>
-      </div>     
+      </div>
 
       <div class="large-4 medium-4 columns">
         <h5>Try one of these buttons:</h5>
-        <p><a href="#" class="button">Simple Button</a><br/>       
+        <p><a href="#" class="button">Simple Button</a><br/>
         <a href="#" class="success button">Success Btn</a><br/>
         <a href="#" class="alert button">Alert Btn</a><br/>
-        <a href="#" class="secondary button">Secondary Btn</a></p>           
+        <a href="#" class="secondary button">Secondary Btn</a></p>
         <div class="callout">
           <h5>So many components, girl!</h5>
           <p>A whole kitchen sink of goodies comes with Foundation. Check out the docs to see them all, along with details on making them your own.</p>
-          <a href="http://foundation.zurb.com/sites/docs/" class="small button">Go to Foundation Docs</a>          
+          <a href="http://foundation.zurb.com/sites/docs/" class="small button">Go to Foundation Docs</a>
         </div>
       </div>
     </div>
-
-    <script src="bower_components/jquery/dist/jquery.js"></script>
-    <script src="bower_components/what-input/dist/what-input.js"></script>
-    <script src="bower_components/foundation-sites/dist/js/foundation.js"></script>
-    <script src="js/app.js"></script>
```

Comment:
Fallback support for older browsers https://github.com/jhabdas/fetch-inject#progressive-enhancement